code/modules/mob/living/living.dm

```diff
@@ -1397,6 +1397,11 @@
 			to_chat(src, span_warning("You don't have the hands for this action!"))
 			return FALSE
 
+	if(!(action_bitflags & ALLOW_PAI) && ispAI(src))
+		if(!(action_bitflags & SILENT_ADJACENCY))
```

Comment:
why check for `SILENT_ADJACENCY`? Since you aren't using procs like `Adjacent()` or `CanReach()` checking for this flag out of those contexts makes no sense

Reply:
Because you can combine different bitflags. Really that flag should just be renamed to SILENT_NOTIFICATION or something similar because all it's doing is ignoring the to_chat messages

Reply:
Why not move this check after this if condition then

https://github.com/tgstation/tgstation/blob/cbaf52bc2de554eb0c44f79bead08eadce4a9f88/code/modules/mob/living/living.dm#L1405

This checks for `SILENT_ADJACENCY` so you don't have to.

Reply:
There are a few edge cases where if a pAI hovers an item it will spam the chat with can't do that messages. It's not a big deal since it only applies to a few niche items but I don't care one way or another.

Applied via https://github.com/tgstation/tgstation/pull/88651/commits/16a17e94f6402a5499886a719b3b3beca4fa2252